### DISCUSSION TITLE
Command Line menu for running any script from repo.

### DISCUSSION THREAD

**Discussion by mon5termatt** [tteck/Proxmox#1874] [Ideas]
Sometimes I don't want to open a web browser. Sometimes I just want to use SSH....

Therefore, I feel like a menu to choose scripts from this repo could be appreciated. 

kinda like how dietpi has a `dietpi-software` command or how Ubuntu asks you if you want to install any software on first install.

**Comment by mon5termatt** [comment]
kinda like https://github.com/tteck/Proxmox/discussions/1829

**Comment by tteck** [comment]
maintenance 👎🏻
